test/rexml/test_functions.rb

```diff
@@ -222,6 +222,22 @@ def test_normalize_space
       assert_equal( [REXML::Comment.new("COMMENT A")], m )
     end
 
+    def test_string_nil_without_context
+      # Reset to default value
+      REXML::Functions.class_variable_set(:@@context, nil)
```

Comment:
We can use `REXML::Functions.context = nil`.

Could you create a setup method and call this? Because this is needed for all tests.

Reply:
All tests in this file? Or the entire test suite?

Reply:
It's enough only for this file.